<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Jonathan MacMillan</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_wrmm">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Write major/minor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_read">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Read base</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_wrtype">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Write base type</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_read">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>62</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_wrtype">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Write type</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>20</width>
      <height>441</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_wrtype">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>20</width>
      <height>441</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_wrmm">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Write major/minor</string>
    </property>
   </widget>
   <widget class="QHexSpinBox" name="sb_wrmaj">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>57</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
   </widget>
   <widget class="QHexSpinBox" name="sb_wrmin">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>57</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBase_lists">
    <property name="title">
     <string>Bases</string>
    </property>
    <addaction name="actionNew_base_list"/>
    <addaction name="actionEdit_base_list"/>
    <addaction name="actionImport_base_list"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionBTBaseUtil_help"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuBase_lists"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionEdit_base_list">
   <property name="text">
    <string>&amp;Edit base list</string>
   </property>
   <property name="toolTip">
    <string>Edit a list of bases</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionImport_base_list">
   <property name="text">
    <string>&amp;Import base list</string>
   </property>
   <property name="toolTip">
    <string>Import a list of bases</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionBTBaseUtil_help">
   <property name="text">
    <string>&amp;BTBaseUtil help</string>
   </property>
   <property name="toolTip">
    <string>Get help for BTBaseUtil</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About BTBaseUtil</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCreate_base_list">
   <property name="text">
    <string>&amp;Create base list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionNew_base_list">
   <property name="text">
    <string>New base list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHexSpinBox</class>
   <extends>QSpinBox</extends>
   <header>QHexSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
